e2e-demo/ldes-client-persistence/README.md

```diff
@@ -58,7 +58,7 @@ To run the test, you need to:
 #### 1. Upload NiFi Workflow
 Log on to the [Apache NiFi user interface](https://localhost:8443/nifi) using the user credentials provided in the `user.env` file.
 
-Once logged in, create a new process group based on the [replicate workflow](./flows/nifi-workflow.json) as specified in [here](../../../support/context/workflow/README.md#creating-a-workflow).
+Once logged in, create a new process group based on the [replicate workflow](./flows/nifi-workflow.json) as specified in [here](https://github.com/Informatievlaanderen/VSDS-LDES-E2E-testing/tree/main/e2e-test/support/context/workflow#creating-a-workflow).
```

Comment:
Hi, nice catch!
We prefer to use relative references because they work both in online in GitHub as well as locally when you pull the sources. So, instead the absolute reference above please use: `../../e2e-test/support/context/simulator-workflow-sink-mongo/README.md`

Reply:
Hello @rorlic ,

Thanks for pointing it out , just modified to relative references ((../../e2e-test/support/context/workflow/README.md#creating-a-workflow).)

Xueying